service/stacks/zephyr/sal_hfp_hf_interface.c

```diff
@@ -179,7 +179,7 @@ static bt_hfp_hf_call_info_t* new_call()
 
 static bt_hfp_hf_call_info_t* find_or_create_call(bt_hfp_hf_connection_t* sal_conn, struct bt_hfp_hf_call* z_context)
 {
-    if (!sal_conn || !sal_conn->calls) {
+    if (!sal_conn || !sal_conn->calls || !z_context) {
         return NULL;
     }
 
```

Comment:
I find that new_call without any pass any argument， would not it build Warning？


static bt_hfp_hf_call_info_t* new_call()